### PR TITLE
e2e: allow running of single node using the e2e app (backport)

### PR DESCRIPTION
**Pull request by cmwaters**
Backport of #6982

### CODE REVIEW

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 
 ### FEATURES
 
+- [\#6982](https://github.com/tendermint/tendermint/pull/6982) tendermint binary has built-in suppport for running the e2e application (with state sync support) (@cmwaters).
```
Comment (tychoish) [inline CHANGELOG_PENDING.md:23]: I'd spell out e2e.

**Review by tychoish — APPROVED** [review]
is there any reason to keep the kv app around if we have this?

**Comment by cmwaters** [comment]
> is there any reason to keep the kv app around if we have this?

We can probably remove the kv app and just have the single app in the tm repo if the rest of the team are on board with it

File: CHANGELOG_PENDING.md (hunk 1 above)
Comment (cmwaters) [inline CHANGELOG_PENDING.md:23]:
```suggestion
- [\#6982](https://github.com/tendermint/tendermint/pull/6982) tendermint binary has built-in suppport for running the end to end application (with state sync support) (@cmwaters).
```

**Pull request merged**